tests/h/views/accounts_test.py

```diff
@@ -574,7 +574,7 @@ def test_get_when_not_logged_in_successful_creates_ActivationEvent(
         ActivationEvent.assert_called_once_with(
             pyramid_request, user_model.get_by_activation.return_value)
 
-    def test_get_when_not_logged_in_successful_notifies(self,
+    def test_get_when_not_logged_in_successful_notifies(self,  # noqa: N803
```

Comment:
I'm not sure what's being disabled here, is there a way to name the rule that's being disabled in the comment?

Reply:
Not as far as I know 😞 - ESLint has human readable slug-like names for its rules which can be used in linter directives. flake8 lacks this.